README.md

```diff
@@ -2,15 +2,22 @@
 <!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
 [![All Contributors](https://img.shields.io/badge/all_contributors-4-orange.svg?style=flat-square)](#contributors-)
 <!-- ALL-CONTRIBUTORS-BADGE:END -->
-A **community-driven**, **open-source**, **Unity3D based** Mount &amp; Blade II : BannerLord module Editor
+
+A **community-driven**, **open-source**, **Unity3D based** Mount &amp; Blade II : BannerLord module Editor.
 
 ## Feature list
 
```

Comment:
* Project file management
User can manage different types of file in their model project easily with access to folder-based file system and version control